articles/tutorials/qft-tutorial.md

```diff
@@ -372,78 +484,98 @@ $$
+The `mutable` keyword prefacing `resultArray` allows the variable to be rebound
+later in the code---for example, when adding our measurement results.
 
-#### Perform measurements in a `for` loop and add results to array
+### Perform measurements in a `for` loop and add results to array
 
-After the Fourier transform operations inside the `using` block, insert the following code:
+After the Fourier transform operations inside the `using` block, insert the
+following code:
 
 ```qsharp
             for(i in IndexRange(qs)) {
                 set resultArray w/= i <- M(qs[i]);
             }
 ```
-The [`IndexRange`](xref:microsoft.quantum.arrays.indexrange) function called on an array (e.g. our array of qubits, `qs`) returns a range over the indices of the array. 
-Here, we use it in our `for` loop to sequentially measure each qubit using the `M(qs[i])` statement.
-Each measured `Result` type (either `Zero` or `One`) is then added to the corresponding index position in `resultArray` with an update-and-reassign statement.
 
-> [!NOTE]
-> The syntax of this statement is unique to Q#, but corresponds to the similar variable reassignment `r[i] <- M(qs[i])` seen in other languages such as F# and R.
+The [`IndexRange`](xref:microsoft.quantum.arrays.indexrange) function called on
+an array (e.g. our array of qubits, `qs`) returns a range over the indices of
+the array. Here, we use it in our `for` loop to sequentially measure each qubit
+using the `M(qs[i])` statement. Each measured `Result` type (either `Zero` or
+`One`) is then added to the corresponding index position in `resultArray` with
+an update-and-reassign statement.
+
+> [!NOTE] 
+> The syntax of this statement is unique to Q#, but corresponds to the
+> similar variable reassignment `resultArray[i] <- M(qs[i])` seen in other languages such
+> as F# and R.
 
 The keyword `set` is always used to reassign variables bound using `mutable`.
 
-#### Return `resultArray`
+### Return `resultArray`
 
-With all three qubits measured and the results added to `resultArray`, we are safe to reset and deallocate the qubits as before.
-After the `using` block's close, insert
+With all three qubits measured and the results added to `resultArray`, we are
+safe to reset and deallocate the qubits as before. After the `using` block's
+close, insert
 
 ```qsharp
+        Message("Post-QFT measurement results [qubit0, qubit1, qubit2]: ");
         return resultArray;
 ```
-to ultimately return the output of our operation. 
+
+to ultimately return and print the output of our operation.
 
 ### Understanding the effects of measurement
 
-Let's change the placement of our `DumpMachine` functions to output the state before and after the measurements.
-The final operation code should look like: 
+Let's change the placement of our `DumpMachine` functions to output the state
+before and after the measurements. The final operation code should look like:
 
 ```qsharp
     operation Perform3QubitQFT() : Result[] {
```

Comment:
Above you had "qubit" in the name of this operation as lowercase; I think having it uppercase as in this example makes a lot of sense, though. Would you be willing to modify the above instances to `Perform3QubitQFT`? Thanks!

Reply:
Done